lib/options.c

```diff
@@ -178,31 +178,6 @@ get_common_opts_from_env (common_opts_t *common_opts)
     if (env_str != NULL)
```

Comment:
I comment here since it seems that Github doesn't allow to comment on the commit message:

s/reallocing/reallocating. Also I think it's more clear if you rephrase as:

"Rather than reallocting 1 new array item for the new argv array, just calloc up to argc count".